Miembros.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Miembros</title>
+        <link id="estilo" rel="stylesheet" type="text/css" href="csses/miembros.css" />
+    </head>
+
+    <body>
+        <header>Dungeon's Hub <img src="img/Dungeon's_Hub.png" height="128" width= "128"></header>
+        <footer>
+            <ul><li><a href="index.html">Inicio</a></li>
+                <li><a href="detalles.html">Detalles</a></li>
+                <li><a href="bocetos.html">Bocetos</a></li>
+                <li><a href="planificacion.html">Planificacion</a></li>
+                <li><a href="contactos.html">Contactos</a></li>
+            </ul>
+        </footer>
+
+        <!--Hoja-->
+            <!--Lateral con referencias-->
+            <h1>Miembros del Proyecto</h1>
+                <ol><li> <a href="#DanielLafuenteBazo">Daniel Lafuente Bazo <img src="img/daniLafu.png" height="64" width= "64"></a></li>
+                    <li> <a href="#HugoVivacioFernandez">Hugo Vivacio Fernandez <img src="img/hugouwu.png" height="64" width= "64"></a> </li>
+                    <li> <a href="#JaimeIsarMuñoz">Jaime Isar Muñoz <img src="img/jimmysaurio.png" height="64" width= "64"></a> </li>
+                    <li> <a href="#AlejandroRodriguezGiner">Alejandro Rodriguez Giner<img src="img/chuvak.png" height="64" width= "64"></a> </li>
+
+                </ol>
+            <!--Lista de miembros del equipo de desarrollo-->
+                <ol><li id="DanielLafuenteBazo">Daniel Lafuente Bazo
```

Comment:
Es más adecuado utilizar un `<section>` por cada miembro. Habéis abusado de `<dl>` porque os faltan los `<dd>`, simplemente los habéis usado para poner negrita y eso se puede simplemente con `<strong>`